ui/components/component-library/text-field-base/text-field-base.test.js

```diff
@@ -246,7 +250,7 @@ describe('TextFieldBase', () => {
     const textFieldBase = getByTestId('text-field-base');
 
     expect(textFieldBase.value).toBe(''); // initial value is empty string
-    fireEvent.change(textFieldBase, { target: { value: 'text value' } });
+    await user.type(textFieldBase, 'text value');
     expect(textFieldBase.value).toBe('text value');
     fireEvent.change(textFieldBase, { target: { value: '' } }); // reset value
     expect(textFieldBase.value).toBe(''); // value is empty string after reset
```

Comment:
Improving unit tests by using the userEvent and render helpers. Still all 🟢 

![Screenshot 2022-11-23 at 2 17 28 PM](https://user-images.githubusercontent.com/8112138/203656716-0a022351-9567-45d6-a791-8d776e324e38.png)